src/dcos_e2e/backends/_existing_cluster.py

```diff
@@ -144,6 +127,46 @@ def __init__(  # pylint: disable=super-init-not-called
             )
             raise ValueError(message)
 
+    def install_dcos_from_url(
+        self,
+        build_artifact: str,
+        extra_config: Dict[str, Any],
+        log_output_live: bool,
+    ) -> None:
+        """
+        Raises:
+            NotImplementedError: Raises `NotImplementedError` because it
+                is assumed that clusters created with the ExistingCluster
+                backend already have an installed instance of DC/OS
+                running on them.
+        """
+        message = (
+            'The ExistingCluster backend does not support the installing '
+            'DC/OS because it is assumed that an instance of DC/OS is '
+            'already installed and running on the cluster.'
+        )
+        raise NotImplementedError(message)
+
+    def install_dcos_from_path(
+        self,
+        build_artifact: Path,
+        extra_config: Dict[str, Any],
+        log_output_live: bool,
+    ) -> None:
+        """
+        Raises:
+            NotImplementedError: Raises `NotImplementedError` because it
+                is assumed that clusters created with the ExistingCluster
+                backend already have an installed instance of DC/OS
+                running on them.
+        """
+        message = (
+            'The ExistingCluster backend does not support installing DC/OS'
```

Comment:
I know this is WIP but I spotted it - DC/OS is repeated "installing DC/OS DC/OS".